bucket/rife-ncnn-vulkan.json

```
@@ -0,0 +1,23 @@
+{
+    "version": "20220330",
+    "description": "RIFE, Real-Time Intermediate Flow Estimation for Video Frame Interpolation implemented with ncnn library",
+    "homepage": "https://github.com/nihui/rife-ncnn-vulkan",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/nihui/rife-ncnn-vulkan/releases/download/20220330/rife-ncnn-vulkan-20220330-windows.zip",
+            "hash": "f79514b3bf3cae59171f4748e4be772ae30942ad377ea574e81d194ba772a3f2",
+            "extract_dir": "rife-ncnn-vulkan-20220330"
+        }
+    },
+    "bin": "rife-ncnn-vulkan.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/nihui/rife-ncnn-vulkan/releases/download/$version/rife-ncnn-vulkan-$version-windows.zip",
+                "extract_dir": "rife-ncnn-vulkan-$version"
```

```suggestion
                "extract_dir": "rife-ncnn-vulkan-$version-windows"
```
Your extract_dir is wrong.